src/view.js

```diff
@@ -15,12 +15,27 @@ const intervals = {
   'major 7th': 11,
 };
 
+const classNames = {
```

Comment:
I am always impressed the way you organize related variables in one object and just grab any of them you need from here, like ` add(
      classNames.centerVertically,
      classNames.flow
    );` this. It is so different that what I have been doing, just lay them out in a row haha.

Reply:
Yea it should be possible! Instead of clicking on a line, click and drag across multiple lines. Though you’ll only be able to comment on lines with changes 😅